src/revpi-factory-reset.sh

```diff
@@ -16,6 +16,15 @@ fi
 export NEWT_COLORS='root=,black entry=white,black'
 
 while [ ! -r /home/pi/.revpi-factory-reset ] ; do
+	if has-hat-eeprom; then
+		#The device is equipped with HAT EEPROM
+		echo -e "For the first booting, the serial number will be retrieved from
```

Comment:
Please show a info dialog with whiptail instead of printing to stdout.

My suggestion for the content would be:
```
The device configuration was detected automatically. Manual configuration is therefore not necessary.
```